src/core/Akka/Configuration/akka.conf

```diff
@@ -49,7 +49,8 @@ akka {
   
   # Fully qualified class name (FQCN) of the very basic logger used on startup and shutdown.
   # You can substitute the logger by supplying an FQCN to a custom class that implements Akka.Event.MinimumLogger
-  stdout-logger-class = "Akka.Event.StandardOutLogger"
+  # Set to null by default so the default logger is used.
+  stdout-logger-class = ""
```

Comment:
Disabled the HOCON loading here, as we always `new` up this class by default if this property is left blank. Removes 1 AOT warning that every Akka.NET application 100% hits and improves type safety.